### PR TITLE
Add documentation (setting up, etc) #5

### LINKED ISSUE #5: Add documentation (setting up, etc)

### PR DESCRIPTION
**Pull request by samsontmr**
Fixes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+
+1. Fork/clone this repository.
+1. Create a new GitHub user account for your bot, e.g. `teammates-bot`.
+1. Create an API access token in this new GitHub account under `Settings -> Personal access tokens`.
+1. Create a new Heroku app.
+1. In the Heroku app settings, click `Reveal Config Vars`. Add the following keys: 
+```
+    GITHUB_API_TOKEN : Your bot's GitHub API access token
+    REGEX_PULL_REQ_TITLE : Python regex to validate the PR title
+    REGEX_PULL_REQ_BODY : Python regex to validate the PR description
```
Comment (damithc) [inline README.md]: Do they have to be python regex?

**Review by damithc — COMMENTED** [review]
May be you can trial it with TEAMMATES first?

File: README.md (hunk 1 above)
Comment (samsontmr) [reply README.md]: Good catch, thanks!

**Comment by samsontmr** [comment]
@damithc Do you mean setting up a TEAMMATES-specific bot server, or just getting TEAMMATES to send this one webhooks?

**Comment by damithc** [comment]
> @damithc Do you mean setting up a TEAMMATES-specific bot server, or just getting TEAMMATES to send this one webhooks?

I guess a free heroku app is not enough for this?

**Comment by samsontmr** [comment]
> I guess a free heroku app is not enough for this?

Hm what do you mean? I'm currently using a free Heroku app. Just that free accounts are limited to 5 apps and dyno use is limited.

**Comment by damithc** [comment]
From what I remember, a free heroku app doesn't have enough juice to stay up 24 hours a day. An app monitoring web hooks needs to be up 24 hours a day.

**Comment by samsontmr** [comment]
From what I know, it goes to sleep when there's no traffic and wakes up when the webhook is triggered.

EDIT: I just tested another of my bots from a while back and it's still responding.

**Comment by damithc** [comment]
Yes, that's how it works but it has a max wakeup time per day, right? I'm sure it also has a min idle period before it goes to sleep?
So if the web hook is triggered often enough it's going to hit max wakeup time per day. Anyway, we can try it with TEAMMATES because if the webhook is only triggered when a new PR is created, the frequency should be low enough for the app to get enough sleep.

**Comment by samsontmr** [comment]
Oh the maximum wakeup time is about 18 hours. Unfortunately, it seems that any modification to a PR triggers the webhook. However, even though the server may be maxed out and unable to respond immediately, it should receive the payload once it comes back online. I think this is still acceptable since it's not a time-critical operation.

**Comment by damithc** [comment]
Let's try it out on TEAMMATES to see how it goes. Let's set up a new heroku app for it so that we can monitor its activity level without being affected by your own dev testing.

**Pull request merged**